sources/web/datalab/polymer/test/item-list-test.ts

```diff
@@ -281,4 +311,113 @@ describe('<item-list>', () => {
     assert(!isSelected(0), 'first item should not be selected when selection is disabled');
   });
 
+  describe('inline details', () => {
+    it('should not create a details element for an item without details', () => {
+      testFixture.inlineDetailsMode = InlineDetailsDisplay.SINGLE_SELECT;
+      const firstRow = getRow(0);
+      const firstRowDetailsContainer = getRowDetailsContainer(0);
+      firstRow.click();
```

Comment:
Don't we need to call `Polymer.dom.flush();` after the click to make sure templates have rendered, since we have some dom-repeats?

Reply:
I'm worried about flakiness, because this call makes sure the render completes before proceeding.